### PR TITLE
Add conclusion

### PR DESCRIPTION
**Pull request by zietzm**

### CODE REVIEW

File: content/02.body.md [hunk 1]
```diff
@@ -234,3 +234,13 @@ Alternatively, if degree is not a desired feature, then the edge prior can be us
 
 
 ## Conclusion
+
+Nodes in biomedical networks can have different degrees, and networks can have very different degree distributions.
+Many edge prediction methods are correlated with degree.
+Degree imbalance, in which edges are distributed unequally between nodes, can lead edge prediction methods to make nonspecific or trivial predictions.
+Sometimes networks faithfully represent all true relationships, but degree biases and other misrepresentations can obscure node degree.
```
Comment (cgreene) [inline content/02.body.md]:
```suggestion
```
I think this paragraph is stronger without this sentence in it. Give it a read without it and see what you think.

File: content/02.body.md [hunk 2]
```diff
@@ -234,3 +234,13 @@ Alternatively, if degree is not a desired feature, then the edge prior can be us
...
+To better understand the effects of degree on an edge prediction task and on edge prediction methods, degree should be considered directly.
```
Comment (cgreene) [inline content/02.body.md]:
```suggestion
Degree should be considered directly in prediction approaches to understand the effects degree has on edge prediction tasks and methods.
```

File: content/02.body.md [hunk 3]
```diff
@@ -234,3 +234,13 @@ Alternatively, if degree is not a desired feature, then the edge prior can be us
...
+We developed a network permutation framework and open source software implementation that quantifies the probability of edge existence due to degree and can be used to assess the fraction of feature performance attributable to degree.
+We demonstrated the superiority of the edge prior over other degree-based features for quantifying the effect of degree on the probability of edge existence.
+We provided an example application of our methods that assessed five edge prediction features' performances on a network reconstruction task by quantifying the methods' reliance on degree and degree's performance on the prediction task.
```
Comment (cgreene) [inline content/02.body.md]:
```suggestion
```
I also think it's stronger without this sentence. Again, see what you think 😁

File: content/02.body.md [hunk 4]
```diff
@@ -234,3 +234,13 @@ Alternatively, if degree is not a desired feature, then the edge prior can be us
...
+Our method provides a context for evaluating edge prediction methods and specific predictions for reliance on degree and, therefore, nonspecificity.
```
Comment (cgreene) [inline content/02.body.md]:
```suggestion
The XSwap methods and software provide a context for evaluating edge prediction methods and specific predictions for reliance on degree and, therefore, nonspecificity.
```

**Review by cgreene — COMMENTED** [review]
Sometimes a good conclusions section gets more general as you get broader. In this case, you might consider moving the specific thing that xswap does to the beginning of this paragraph and then rearrange so it gets broader (starts about xswap, then more broadly about the context that this has in biomedicine. You might try re-arranging things this way to see if you like it.

There were also two sentences in there that, in this current framing, I didn't think strengthened things.

**Comment by zietzm** [comment]
I agree that the sentences you suggest removing do not help the conclusion.

> You might try re-arranging things this way to see if you like it.

This sounds like a good idea. I'll try moving the sentences around and maybe it'll read a bit easier.

**Review by cgreene — APPROVED** [review]
Woo! 👍

**Pull request merged**